app-text/xlreader/xlreader-0.9.0.recipe

```diff
@@ -0,0 +1,40 @@
+SUMMARY="Excel document reader"
+DESCRIPTION="Read data from Excel 97+ spread sheets without Microsoft! \
+Provides an API to write applications that read Excel documents. The xlreader \
+tool converts data to tab delimited, CSV or SQL inserts. Written in C. Based on \
+the the Java version by Andrew Khan."
+HOMEPAGE="http://giffin.org/xlreader"
+COPYRIGHT="2001 David Giffin"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/xlreader/xlreader-$portVersion.tgz"
+CHECKSUM_SHA256="53d556a77d67b3d6e8a4919a7753c019fc5395b8e2f5e33eb324dcf7c075d7fa"
+SOURCE_DIR="xlreader-$portVersion"
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	xlreader = $portVersion
+	cmd:xlreader = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
```

Comment:
Please sort alphabetically the 2 cmd: lines in BUILD_PREREQUIRES.

Hmm, if you pick the suggestion for the modified README, then it would be nice to also include a `cmd:sed` in BUILD_PREREQUIRES.